packages/webcert/src/feature/certificate/Category/Category.tsx

```diff
@@ -5,11 +5,14 @@ import CategoryHeader from './CategoryHeader'
 import CategoryTitle from './CategoryTitle'
 import { Accordion, Expandable } from '@frontend/common'
 import _ from 'lodash'
+import { css } from 'styled-components'
 
 interface CategoryProps {
   id: string
 }
 
+const wrapperStyles = css``
```

Comment:
Wouldn't it be better to make wrapperStyles optional?

Reply:
Yes, but I thought the component was from a component library and that we cannot change it?

Reply:
No there are no components from the component library at the moment. It was released only a week ago. We use the UX-library styling which is implemented by adding classnames